keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -77,10 +78,10 @@ Components in Kubernetes can similarly be configured by multiple entities and pr
 
 ### Goals
 
-* Add support for a "--config-dir" flag to the kubelet to allow users to specify a drop-in directory, which will override the configuration for the Kubelet located at `/etc/kubernetes/kubelet.conf`
+* Add support for a `--config-dir` flag to the kubelet to allow users to specify a drop-in directory, which will override the configuration for the Kubelet located at `/etc/kubernetes/kubelet.conf`
 * Extend kubelet configuration parsing code to handle files in the drop-in directory.
 * Define Kubernetes best-practices for configuration definitions, similarly to [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md). This is intended for other maintainers who would wish to setup a configuration object that works well with drop-in directories.
-* As a goal for beta, Add ability to easily view the effective configuration that is being used by kubelet.
```

Comment:
Why this goal was removed? Can we make it a goal without the attachment to a specific milestone